src/Animated.test.js

```diff
@@ -0,0 +1,106 @@
+import Animated, { Easing } from './Animated';
+import ReanimatedModule from './ReanimatedModule';
+import AnimatedNode from './core/AnimatedNode';
+
+jest.mock('./ReanimatedEventEmitter');
+jest.mock('./ReanimatedModule');
+jest.mock('./derived/evaluateOnce');
+
+const { Value, timing, spring, decay } = Animated;
+describe('Reanimated backward compatible API', () => {
+  beforeEach(() => {
+    let nodesCreated = 0;
+    ReanimatedModule.createNode = () => nodesCreated++;
+    ReanimatedModule.dropNode = () => nodesCreated--;
+    ReanimatedModule.getNumberOfNodes = () => nodesCreated;
+  });
+
+  const checkIfAttachAndDetachNodesProperly = jest.fn(animation => {
```

Comment:
I don't think there is a point in using `jest.fn` for this method. `js.fn` should only be used for "stubbed" or "mocked" methods that you need later to assert on if they got called and with what arguments etc. This is not the case here